configs/openSUSE/dbus-services.toml

```diff
@@ -66,10 +66,11 @@ hash = "00d25f898cf4c78cf45d68000ee00be27b0aa15b7ebe22c4ba86bc1b5f33b898"
 [[FileDigestGroup]]
 package = "gdm"
 type = "dbus"
-note = "legacy: not audited"
-nodigests = [
-    "/etc/dbus-1/system.d/gdm.conf",
-]
+bug = "bsc#1204052"
+[[FileDigestGroup.digests]]
+path = "/usr/share/dbus-1/system.d/gdm.conf"
+digester = "xml"
+hash = "e5ce53f9dd19a1d3a3dd4f221b3b1c3a90448c95cceddc4386ca49a999977a36"
```

Comment:
It seems you've simply used sha256sum to calculate this digest. You need to use the helper script from the rpmlint repository `tools/get_whitelisting_digest.py` to get the correct digest for the XML filter.